modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/DefinitionOfTerms.tsx

```diff
@@ -17,41 +17,32 @@ import ClayPanel from '@clayui/panel';
 // @ts-ignore
 
 import {FrontendDataSet} from '@liferay/frontend-data-set-web';
-
-// @ts-ignore
-
-import {render} from '@liferay/frontend-js-react-web';
 import {
 	API,
 	AutoComplete,
 	onActionDropdownItemClick,
 } from '@liferay/object-js-components-web';
 import {createResourceURL, fetch} from 'frontend-js-web';
-import React, {useEffect, useState} from 'react';
+import React, {useEffect, useMemo, useState} from 'react';
 
 const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
 
-export function DefinitionOfTerms({baseResourceURL}: IProps) {
-	const [objectDefinitions, setObjectDefinitions] = useState<
-		ObjectDefinition[]
-	>();
-	const [selectedEntity, setSelectedEntity] = useState<ObjectDefinition>();
-	const [query, setQuery] = useState<string>('');
-
-	const dataSetProps: IDataSetProps = {
-		id: 'tableTest',
-		items: [],
+const getDataSetProps = (items: Item[]) => {
+	return {
+		id: 'DefinitionOfTermsTable',
+		items,
 		itemsActions: [
 			{
 				href: 'copyObjectFieldTerm',
 				id: 'copyObjectFieldTerm',
-				label: 'copy',
+				label: Liferay.Language.get('copy'),
 				target: 'event',
-			} as any,
+			},
 		],
 		namespace: '',
 
 		onActionDropdownItemClick,
+
 		pageSize: 5,
```

Comment:
Hi @rebsilva. Do we need this empty line?

Reply:
I think the `yarn format ` made this change but I will confirm if I can remove this, thanks Alininha! :heart: